pycbc/inference/recal.py

```diff
@@ -0,0 +1,102 @@
+# Module to hold recalibration stuff used during parameter estimation
+# This will draw heavily from pycbc_adjust_strain and cal.py which
+# are part of Chris Biwer's pycbc-cal repository
+
+import numpy
+from scipy.interpolate import UnivariateSpline
+from pycbc.types import FrequencySeries
+
+class Recalibrate(object):
+    """ Class for adjusting time-varying calibration parameters.
+    """
+
+    def __init__(self, calib_dict=None):
```

Comment:
This is more style thing but it might be better to pass the strain here as well and store that as ``self.strain`` so you don't have to keep passing it to the ``adjust`` function.